test/fixtures/linter/autofix/ansi/016_index_error_with_jinja_if/before.sql

```diff
@@ -0,0 +1,16 @@
+-- This file combines product data from individual brands into a staging table
+{% set products =  [
+  'table1',
+  'table2'] %}
+
+{% for product in products %}
+SELECT
+  brand,
+  country_code,
+  category,
+  name,
+  id
+FROM
+  product
```

Comment:
This really should be this as per above:

```suggestion
  {{ product }}
```

However when I test this with `{% if not loop.last -%} UNION ALL {%- endif %}` syntax it _dedents_ this as part of the fix (so fixes it to `{{ product}}` instead of `••••{{ product }}`(where `•` represents a space) but it correctly indents it with `{% if not loop.last %} UNION ALL {% endif %}` so a bit confused as to why that happens. Any ideas?